morpheushelper/cogs/automod.py

```diff
@@ -176,7 +176,7 @@ async def autokick_role(self, ctx: Context, *, role: Role):
             title=translations.autokick, description=translations.autokick_role_configured, colour=Colours.AutoMod
         )
         await ctx.send(embed=embed)
-        await send_to_changelog(ctx.guild, translations.f_log_autokick_role_configured(role.name, role.id))
+        await send_to_changelog(ctx.guild, translations.f_log_autokick_role_configured(role.mention, role.id))
```

Comment:
Didn't we say, that we don't want any role mentions in the changelog,  because even the roll is deleted, we can't see which roles was configured.